app/views/profiles/edit.html.erb

```diff
@@ -43,6 +43,9 @@
   <fieldset>
     <legend><%= t ".home location" -%></legend>
     <p id="home_message" class="text-muted m-0<% if current_user.home_location? %> invisible<% end %>"><%= t ".no home location" %></p>
+    <p class="text-muted m-0<% if current_user.home_location? %> invisible<% end %>">
+       <b><%= t(".home_location_warning") %></b>
```

Comment:
This change will no longer work as you've removed the `home_location_warning` string from the locale file - you were asked to move this to a separate PR which would have required moving both this and the string but instead you just removed the string from this PR and left this.